src/lib/y2storage/boot_requirements_strategies/zipl.rb

```diff
@@ -48,12 +64,34 @@ def zipl_partition_missing?
         free_mountpoint?("/boot/zipl")
       end
 
+      # @return [VolumeSpecification]
+      def zipl_volume
+        return @zipl_volume unless @zipl_volume.nil?
+
+        @zipl_volume = VolumeSpecification.new({})
+        @zipl_volume.mount_point = "/boot/zipl"
+        @zipl_volume.fs_types = [Filesystems::Type::EXT2]
+        @zipl_volume.fs_type = Filesystems::Type::EXT2
+        @zipl_volume.min_size = DiskSize.MiB(100)
+        @zipl_volume.desired_size = DiskSize.MiB(200)
+        @zipl_volume.max_size = DiskSize.GiB(1)
+        @zipl_volume
+      end
+
+      # @return [Planned::Partition]
       def zipl_partition(target)
-        vol = Planned::Partition.new("/boot/zipl", Filesystems::Type::EXT2)
-        vol.disk = boot_disk.name
-        vol.min_size = target == :min ? DiskSize.MiB(100) : DiskSize.MiB(200)
-        vol.max_size = DiskSize.GiB(1)
-        vol
+        planned_partition = create_planned_partition(zipl_volume, target)
+        planned_partition.disk = boot_disk.name
+        planned_partition
+      end
+
+      # Specific error when the boot disk is not valid for booting
+      #
+      # @return [SetupError]
+      def unsupported_boot_disk_error
+        # TRANSLATORS: error message
+        error_message = _("Current boot disk cannot be used for booting")
```

Comment:
We need a better message here, specially taking into account that this (as all boot-related errors) could be a false positive.

Something like "Looks like the system is going to be installed on a FBA or LDL device. Booting from such device is not supported". And yes, I know the message is not agnostic enough if we change the conditions in `supported_boot_disk?` but sometimes you need to duplicate that to have human-readable messages.